contrib/python-tldextract/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "python-tldextract"
+pkgver = "5.1.2"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+makedepends = [
+    "python-filelock",
+    "python-idna",
+    "python-requests",
+    "python-requests-file",
+]
```

Comment:
host

Reply:
or just remove them and --skip-dependency-check in makebuildargs

Reply:
or whatever it was called anyway

Reply:
yea would prolly prefer --skip-dependency-check

Reply:
ok

Reply:
seemingly no manual dependency skipping was actually needed

Reply:
yes, adding to checkdepends is the same as host so they are present in the end :)  

but with !check it would fail

Reply:
actually no, theyre installed anyway